xbmc/interfaces/legacy/Dialog.h

```diff
@@ -96,29 +96,31 @@ namespace XBMCAddon
        * heading        : string or unicode - dialog heading.\n
        * list           : string list - list of items.\n
        * autoclose      : [opt] integer - milliseconds to autoclose dialog. (default=do not autoclose)\n
+       * preselect      : [opt] integer - index of preselected item. (default=no preselected item)\n
        * \n
        * *Note, Returns the position of the highlighted item as an integer.\n
        * \n
        * example:\n
        *   - dialog = xbmcgui.Dialog()\n
        *   - ret = dialog.select('Choose a playlist', ['Playlist #1', 'Playlist #2, 'Playlist #3'])\n\n
        */
-      int select(const String& heading, const std::vector<String>& list, int autoclose=0);
+      int select(const String& heading, const std::vector<String>& list, int autoclose=0, int preselect=-1);
 
       /**
        * multiselect(heading, list [,autoclose]) -- Show a multi-select dialog.
        *
```

Comment:
This needs updating as well. Perhaps also add example how that preselect list should look like